modules/ROOT/pages/jakarta-ee10-diff.adoc

```diff
@@ -51,13 +51,13 @@ The following Jakarta EE 10 Liberty features do not expose the XML Binding 4.0 A
 - feature:connectors-2.1[display=Jakarta Connectors 2.1 ] (`connectors-2.1`) 
 - feature:persistence-3.1[display=Jakarta Persistence 3.1] (`persistence-3.1`) 
 - feature:persistenceContainer-3.1[display=Jakarta Persistence Container 3.1] (`persistenceContainer-3.1`) 
-- feature:webProfile-10[display=Jakarta EE Web Profile 10.0] (`webProfile-10`) 
+- feature:webProfile-10[display=Jakarta EE Web Profile 10.0] (`webProfile-10.0`) 
 
 For Open Liberty applications that require the XML Binding APIs, you must now add the feature:xmlBinding-4.0[display=Jakarta XML Binding 4.0] (`xmlBinding-4.0`) Liberty feature to your `server.xml` file.
 
 === Connectors Inbound Security replacement
 
-In Jakarta EE 10, the combination of the Jakarta Connectors 2.1 and feature:appSecurity-5.0[display=Application Security (Jakarta Security) 5.0] (`appSecurity-5.0`) Liberty features replaces what the feature:connectorsInboundSecurity-2.0[display=Jakarta Connectors Inbound Security 2.0] (`connectorsInboundSecurity-2.0`) Liberty feature provided in Jakarta EE 9.1.
+In Jakarta EE 10, the combination of the Jakarta Connectors 2.1 (`connectors-2.1`) and feature:appSecurity-5.0[display=Application Security 5.0] (`appSecurity-5.0`) Liberty features replaces what the feature:connectorsInboundSecurity-2.0[display=Jakarta Connectors Inbound Security 2.0] (`connectorsInboundSecurity-2.0`) Liberty feature provided in Jakarta EE 9.1.
```

Comment:
for appSecurity- we have been using `Application Security (Jakarta Security) *.0` for the text name because that is how it shows up on the autogenerated [feature page](https://openliberty.io/docs/latest/reference/feature/appSecurity-4.0.html). But leaving off `(Jakarta Security)` is a more readable solution. And including 2 version numbers for one feature could be confusing.